commerce-cloud-shop-api/1.0/modules/ROOT/pages/shop-api-connector-troubleshoot.adoc

```diff
@@ -0,0 +1,61 @@
+= Commerce Cloud B2C Shop Connector - Troubleshoot - Mule 4
+
+These are some common troubleshooting suggestions for Anypoint Connector for Salesforce Commerce Cloud B2C Shop APIs (B2C Shop Connector).
+
+*400 Bad Request*
+
+A `400 Bad Request` error response can occur if:
+
+* The URI parameters such as Organization Id, Site Id and Version parameter not passed with the proper value.
+* The above parameter passed with wrong value
+* The above parameter passed with an extra space
+
+== Solution
+
+Correct and pass the proper values.
+
+*401 Unauthorized*
+
+A `401 Unauthorized` error response can occur if:
+
+* The client ID used in the configuration is incorrect.
+* The client secret used in the configuration is incorrect.
+
+== Solution
+
+* Correct and pass the proper values for the client ID and client secret.
+* If the Client ID and client secret are both correct and you are still getting the error, contact the Commerce Cloud team.
+
+*403 Forbidden*
+
+A `403 Forbidden` error response can occur if the scopes used in the configuration are incorrect.
+
+== Solution
+
+Correct and pass the proper value for the scopes.
+
+*405 Method Not Allowed*
+
+A `405 Method Not Allowed` error can occur if a resource that previously supported HTTP POST got removed and had its HTTP GET method deprecated.
```

Comment:
```suggestion
A `405 Method Not Allowed` error can occur if a resource that previously supported HTTP POST no longer does and its HTTP GET method becomes deprecated.
```
Are these two separate conditions or one condition as a result of the other? Clarify.